src/Loco.php

```diff
@@ -195,7 +195,13 @@ public function import(MessageCatalogueInterface $catalogue)
         $locale = $catalogue->getLocale();
         foreach ($this->projects as $project) {
             foreach ($project->getDomains() as $domain) {
-                $data = XliffConverter::catalogueToContent($catalogue, $domain);
+                $options = [];
+
+                if ('id' === $project->getIndexParameter()) {
+                    $options['default_locale'] = 'en';
```

Comment:
As per https://github.com/php-translation/symfony-bundle/issues/280#issuecomment-463679141, should this be "en" or a non-existing language code?

Reply:
IMO, setting the `default_locale` to `en` is confusing. We should use an non-existing language code like `x-id` or something like this.

See https://tools.ietf.org/html/bcp47 & search for "Private use registry values"

Reply:
(this is the solution recommended by @timwhitlock here https://github.com/php-translation/symfony-bundle/issues/280#issuecomment-463679141)

Reply:
Alright, I'll change it to `x-id` then unless you prefer something else.